iron-autogrow-textarea.html

```diff
@@ -78,7 +78,6 @@
         autocomplete$="[[autocomplete]]"
         autofocus$="[[autofocus]]"
         inputmode$="[[inputmode]]"
-        name$="[[name]]"
```

Comment:
Ok. The coordination with the form behavior and elements that compose form elements seem complicated. It would be nice to see if there's a cleaner way, but maybe don't need to do that for this PR for now. I trust your best judgement.